locales/en/app.ftl

```diff
@@ -762,3 +762,7 @@ ecosystem-promo-body =
 promo-ecosystem-cta = See All Products
 
 steps-to-resolve-headline = Steps to resolve this breach
+
+vpn-promo-headline = Now’s the time to boost your safety online.
+vpn-promo-copy = { -brand-Mozilla }’s Virtual Private Network helps shield your internet connection from hackers and spies. 
```

Comment:
A while back, there was an unwritten rule using genitive on brand names. Just to double check, is this copy coming from the Content Team?

Reply:
@flodolo Double-checking!